src/parser/parser_verification.js

```diff
@@ -69,8 +69,13 @@ function verifyRequiredAttributes(node, emit) {
  */
 function verifyRequiredSubElements(node, emit, isAdInline) {
   const required = requiredValues[node.nodeName];
-
-  if (!required) {
+  // Wrapper child elements have no required subElement. (Only InLine does)
+  if (
+    !required ||
+    (!isAdInline &&
```

Comment:
you should only depend on `isAdInline` here. It is confusing that you have to check for the node name.

Reply:
As discussed i will use a better naming 😉